grails-app/conf/au/org/emii/aatams/VisibilityControlFilters.groovy

```diff
@@ -36,15 +36,16 @@ class VisibilityControlFilters
         {
             after = { model ->
 
-                def instanceName = "${controllerName}Instance"
-                def instance = model[instanceName]
+                if (model) {
```

Comment:
There are a couple of cases, the trick is that these filters are performed *after* the action itself. So there is no model after the 'delete' action (as it has been deleted), and there is no model after the 'update' action (it does a redirect with an `id` not with a model object).